openrewrite/src/javascript/parser.ts

```diff
@@ -549,7 +549,15 @@ export class JavaScriptParserVisitor {
     }
 
     visitTypeParameter(node: ts.TypeParameterDeclaration) {
-        return this.visitUnknown(node);
+        return new J.TypeParameter(
+            randomId(),
+            this.prefix(node),
+            Markers.EMPTY,
+            [],
+            [],
+            this.visit(node.name),
+            null // fix me. this should support complex generics
```

Comment:
I think if we don't map it yet, we should return `J.Unknown`. I typically try to catch these cases with an `if` at the beginning.

Reply:
fixed